### ISSUE TITLE
improve start/stop/down operations

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/mcloud-ios#64]
Mac OS always delivers surprises with launchctl service so let's split two approaches and:
1. during setup we can suggest to configure service auto startup and do launchctl load of the zebrunner.plist
2. `./zebrunner.sh start` should start as is service-by-service using current user explicitly
3. `./zebrunner.sh down` should stop/kill processes and suggest to unload services if enabled
4. think about adding new `./zebrunner.sh unload` for `launchctl unload ...` operation as well

**Comment by vdelendik** [comment]
significantly reorganized services manipulation operations:
1. added `status` call which for now determine status of the syncZebrunner launch agent services and echo TODO
2. added `load/unload` calls which identify current syncZebrunner state and do load/unload
-> during the testing run above command twice to see warn, that services already loaded or not loaded
3. `start` operation to load for syncZebrunner and do kickstart of the service asap
-> do start after unload and analyze `./logs/agent.log` in parallel. devices syncup operation should be started asap (earlier we had to wait ~15sec)
4. `stop` also verify state of the syncZebrunner and echo warning that services will be restarted automatically because sync is loaded
5. `down` do force stop and unload of syncZebrunner

**Comment by dhreben** [comment]
Verified.Closed.
Opened separate ticket https://github.com/zebrunner/mcloud-ios/issues/81
